### PR TITLE
Add users.users

### PR DESCRIPTION
**Pull request by bojidar-bg**
#4

### CODE REVIEW

File: lib/methods/users.js [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
+/**
+ * Gets all the users on the site or returns the users identified in [ids].
+ *
+ * @param {Object} criteria
+ * @param {Function} callback return results
+ * @param {Array} ids collection of IDs
+ * @api public
+ */
+function users(criteria, callback, ids) {
+	ids = ids || [];
```
Comment (Swaagie) [inline lib/methods/users.js]: could you change this to 2 space indent please

File: lib/methods/users.js [hunk 2]
```diff
@@ -6,6 +6,18 @@
 var query = require('../query')
   , Logger = require('devnull')
   , log = new Logger({ timestamp: false });
+/**
```
Comment (Swaagie) [inline lib/methods/users.js]: could use enter above to let the code breath

**Comment by Swaagie** [comment]
minor nitpicks, also would like to have a test for this :)

**Comment by Trott** [comment]
Addressed nits and landed as 70f2539b52b965d896038f98b76992f551cceb9c.

Added minimal tests in 44ce58cdd916c5791c8c51add8331f5618cf9e15.

Added minimal docs in de89a04ca773f2a8ad0d7709df8c54037e8f1cea .

**Pull request closed without merging**